src/Extensions/Compatibility/OmexCompatibilityIntializerService.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using Microsoft.Omex.Extensions.Abstractions.Activities;
+
+namespace Microsoft.Omex.Extensions.Compatibility
+{
+	internal sealed class OmexCompatibilityIntializerService : IHostedService
```

Comment:
Please add comments for the class

Reply:
It's an internal class it won't generate any documentation